### ISSUE TITLE
Errors appearing with tuples

### ISSUE THREAD

**Issue by oxysoft** [JetBrains/resharper-unity#911]
These errors are not present in official release Rider 2018.2.3. I've noticed this problem in Rider Nightly for for a few weeks now. It is still happening so far in Build #RD-182.4231.496.

Anywhere tuples are used, there is the following error:
```
Type System.ValueTuple. is not resolved
```

I have another problem too, which could possibly related. I started getting a lot of errors which kind of look like types are being duplicated or registered twice.

![image](https://user-images.githubusercontent.com/3945277/48664990-235a6500-ea75-11e8-9c9f-02266606dc94.png)

Not really sure what's going on, I can try to provide more information as requested.

**Comment by van800** [comment]
Have you tried to Build in Rider? If you don't see the same issue on Build it must be some caches corruption so call File | Invalidate Caches/Restart (https://www.jetbrains.com/help/rider/Cleaning_System_Cache.html)

**Comment by oxysoft** [comment]
I tried already, same problem still.

**Comment by van800** [comment]
So, does it compile in Unity and|or Rider?
Could you please share at least a piece of code. Having a small repro
solution would be even better. Thank you!

**Comment by oxysoft** [comment]
If I try to build in Rider (note this is the first time I'm building a Unity project in Rider, so I could be doing it wrong) I get several instances of this:

```
[CS8107] Feature 'default literal' is not available in C# 7.0. Please use language version 7.1 or greater.
```

I should mention, I am using the Incremental Compiler package from Unity for C# 7.0 support, not the support out of the box in newer versions.

Unsure if I can get a small project to reproduce it, we'll see.

**Comment by van800** [comment]
Please check what do you have in LangVersion tag in csproj? Btw, what is your OS?

**Comment by van800** [comment]
I think this is fixed in 18.3 eap/nightly builds, could you please try one of those.
I recommend using https://www.jetbrains.com/toolbox/app/
